### PR TITLE
Closes #137 UXD: New pixel-perfection.md

### LINKED ISSUE #137: UXD: New pixel-perfection.md

### PR DESCRIPTION
**Pull request by danrichman**
Closes #137

### CODE REVIEW

File: ux-dev/pixel-perfection.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+Pixel perfection is regarded by some to be controversial (either [meaningless](https://www.designernews.co/comments/222544) or [madness](http://blog.kylegawley.com/pixel-perfection-is-madness/)) in the sense that designs [are not rendered the same](http://dowebsitesneedtolookexactlythesameineverybrowser.com/) in all devices. Nevertheless, many clients and their brands require a very high level of UX precision. With the right planning and process, it is possible to approach pixel perfection, for all practical purposes.
+
+#### Why Strive for Pixel perfection?
+
+* Brand identity preservation
+* Avoid compounding layout errors
+* Reliable and reusable modularity
+* Improved UX
+* Improved design quality and aesthetics.
```
Comment (monicawheeler) [inline ux-dev/pixel-perfection.md]: Omit the period here :)

File: ux-dev/pixel-perfection.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+### Mockups are not always correct.
+
+At times, designs may have errors or deviate slightly from existing conventions, grids and standards. Use your best judgement to determine when it makes sense to adhere to, or deviate from, mockups. When it doubt, seek design direction (cc'ing project managers) and assess the risks at scale. 
+
+### Consider unintended consequences
+
+There may be issues that result from standardizing distances between baselines versus distance between modules or character maps. For instance, `line-height` may not be static across locales. Or two different modules may introduce different vertical flows. Do your best to consider these unintended consequences.
+
+### Be flexible and weight risks
```
Comment (monicawheeler) [inline ux-dev/pixel-perfection.md]: > weight
Should be weigh?

File: ux-dev/pixel-perfection.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+## Common pitfalls
+
+#### Over-engineering
+
+There are times you may be tempted to over-engineer a solution in order to achieve pixel perfection. However, this is in and of itself a risk. Err on the side of caution. Achieving pixel perfection is not always worth the risks if it undermines the existing libraries or design systems that allow a site to scale. Keep your solutions simple.
+
+
+#### Pixel-nudging
+
```
Comment (jsit) [inline ux-dev/pixel-perfection.md]: Maybe discuss approaches to pixel nudging? `transform` vs `position: relative` vs `margin`?

Comment (danrichman) [reply ux-dev/pixel-perfection.md]: @jsit Can you elaborate? I didn't want to get into the academic/technical details of pixel-nudging. Was just about recognizing when design systems or mockups aren't synched up.

**Pull request merged**